atomics/include/desul/atomics/Fetch_Op_Generic.hpp

```diff
@@ -78,8 +78,11 @@ DESUL_IMPL_ATOMIC_FETCH_OP_SHIFT_HOST_AND_DEVICE(rshift)
 #undef DESUL_IMPL_ATOMIC_FETCH_OP_SHIFT_HOST_AND_DEVICE
 #undef DESUL_IMPL_ATOMIC_FETCH_OP_SHIFT
 
-// NOTE: using atomic_oper_fetch in the fallback implementation of atomic_store to avoid
-// reading potentially uninitialized values which would yield undefined behavior.
+// NOTE: We would want to use atomic_oper_fetch in the fallback implementation of
+// atomic_store to avoid reading potentially uninitialized values which would yield
+// undefined behavior. As atomic_oper_fetch is not implemented, we have specializations
+// of the lock based fetch_oper for _store_fetch_operator that uses a default
+// constructed value instead of reading from a potentially uninitialized address.
```

Comment:
I wouldn't change anything here. This comment just says that semantically it makes more sense to use `oper_fetch` than `fetch_oper` since the former reads a value first while the latter doesn't. 
If `oper_fetch` ends up using `fetch_oper` nonetheless isn't of concern here but rather at the place where we ensure to skip fetching.